bin/acm

```diff
@@ -5,23 +5,93 @@ require 'thor'
 
 class AkamaiCloudletManagerCli < Thor
 
-  desc "clone_version", "Clones the current version"
+  desc "get_policy_version", "Get policy versions"
   long_desc <<-LONGDESC
-  Clones the current activated version. All new updates are performed in cloned version
+  Get the policy versions.
+    With --policy_id, policy on which version to be cloned
     LONGDESC
-  def clone_version
-    puts "clones the version"
+  option :policy_id, required: true
+  def get_policy_version
+    puts "Policy version:"
+    policy_version = AkamaiCloudletManager::PolicyVersion.new({policy_id: options[:policy_id] })
+    puts policy_version.versions
```

Comment:
should it be just (without puts) `policy_version.versions`

Reply:
I would like to show all versions to user, without puts, not output will be shown